pykokkos/core/visitors/workunit_visitor.py

```diff
@@ -33,7 +33,7 @@ def visit_FunctionDef(self, node: ast.FunctionDef) -> Union[str, Tuple[str, cppa
         else:
             operation: Optional[str] = self.get_operation_type(node)
             if operation is None:
-                self.error(node.args, "Incorrect types in workunit definition")
+                self.error(node.args, "Incorrect types in workunit definition: " + node.name)
```

Comment:
use an f-string instead of  `+ node.name`

Reply:
Reverting this change, this was just to see where my implementation was breaking.